developer_manual/general/devenv.rst

```diff
@@ -19,7 +19,7 @@ Get the source
 
 There are two ways to obtain ownCloud sources: 
 
-* Using the `stable version <http://doc.owncloud.org/server/8.2/admin_manual/#installation>`_
+* Using the `stable version <http://doc.owncloud.org/server/8.0/admin_manual/#installation>`_
```

Comment:
@MorrisJobke Was this reference to "8.2" really supposed to go backwards to "8.0"?